parser/skip_valid_block.go

```diff
@@ -0,0 +1,16 @@
+package parser
+
+import "strings"
+
```

Comment:
// skipValidBlocks defines a list of valid blocks to be skipped during initialization.
// This string is split by newline characters, with each trimmed block name added to the SkipValidBlocks mapping.